src/main/java/io/github/thebusybiscuit/slimefun4/api/events/TalismanActivatedEvent.java

```diff
@@ -0,0 +1,84 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.magical.talismans.Talisman;
+
+/**
+ * 
+ * This {@link PlayerEvent} is called when a {@link Player} activates a {@link Talisman}
+ * 
+ * @author cworldstar
+ *
+ */
+
+public class TalismanActivatedEvent extends PlayerEvent implements Cancellable {
+
+	private static HandlerList Handlers = new HandlerList();
+	private Talisman talisman;
+	private ItemStack talismanItemStack;
+	private boolean cancelled = false;
+	
+	
+	/**
+	 * 
+	 * @param player
+	 * 		The {@link Player} who activated the talisman.
+	 * 
+	 * @param talisman
+	 * 		The {@link Talisman} that was activated.
+	 * 
+	 * @param talismanItem
+	 * 		The {@link ItemStack} corresponding to the Talisman.
+	 */
+	@ParametersAreNonnullByDefault
+	public TalismanActivatedEvent(Player player, Talisman talisman, ItemStack talismanItem) {
+		super(player);
+		this.talisman = talisman;
+		this.talismanItemStack = talismanItem;
+	}
+	/**
+	 * 
+	 * @return The {@link Talisman} used.
+	 *
+	 */
+	@Nonnull

```

Comment:
According to the [style preferences](https://github.com/Slimefun/Slimefun4/blob/dc22e6658990dd7912dabbbfb79a0a24050eed74/CONTRIBUTING.md#7-style-preferences), annotations that target the return type should be inline. Same for line 60.
```suggestion
	public @Nonnull Talisman getTalisman() {
```